.github/workflows/test.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 Hewlett Packard Enterprise Development LP
+name: Test
+
+on: [push, pull_request]
```

Comment:
This will mean that the workflow is execute twice on each push to a PR branch. Should limit the `push` trigger to the main branch